src/test/steps/you-need-to-review-your-application.ts

```diff
@@ -0,0 +1,52 @@
+import { Checkbox } from '../../main/app/case/case';
+import { ApplicationType, DivorceOrDissolution, Gender, YesOrNo } from '../../main/app/case/definition';
+
+import { iSetTheUsersCaseTo } from './common';
+
+const caseData = {
+  applicant1HelpWithFeesRefNo: 'HWF-ABC-123',
+  certificateInEnglish: YesOrNo.NO,
+  hasCertificate: YesOrNo.YES,
+  applicant1AlreadyAppliedForHelpPaying: YesOrNo.YES,
+  inTheUk: YesOrNo.NO,
+  certifiedTranslation: YesOrNo.YES,
+  applicant1HelpPayingNeeded: YesOrNo.YES,
+  ceremonyCountry: 'Mozambique',
+  'relationshipDate-day': 31,
+  'relationshipDate-month': 12,
+  'relationshipDate-year': 1999,
+  ceremonyPlace: 'Maputo',
+};
```

Comment:
caseData might need to be updated after my recent commit - https://github.com/hmcts/nfdiv-frontend/commit/43ceab0275d5cb9725301b524cf4dfa9bcf8c694

I've made it so the application can only be sent for review once all the required fields are filled in

Reply:
I seem to be getting a Green build right now - have you merged that commit?

Reply:
Have updated the PR to use your full case fixture from your PR